File: Java/TurnTicketDispenser/src/test/java/tddmicroexercises/turnticketdispenser/TicketDispenserTest.java
package tddmicroexercises.turnticketdispenser;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

class TicketDispenserTest {

    @Test
    void foo() {
        TicketDispenser dispenser = new TicketDispenser();
        TurnTicket ticket = dispenser.getTurnTicket();
        assertEquals(-1, ticket.getTurnNumber());
    }

}

File: TypeScript/turn-ticket-dispenser-tests/ticket-dispenser-test.ts
import { expect } from 'chai';
import 'mocha';
import TicketDispenser from '../turn-ticket-dispenser/ticket-dispenser';

describe('Turn Ticket Dispenser', () => {

	describe('TurnTicketDispenser', () => {

		it('foo', () => {
			const dispenser = new TicketDispenser();
			const ticket = dispenser.getTurnTicket();
			expect(ticket.getTurnNumber()).to.eql(-1);
		});

	});

});
